symengine/cwrapper.h

```diff
@@ -84,6 +84,19 @@ void basic_free_stack(basic s);
 basic_struct* basic_new_heap();
 void basic_free_heap(basic_struct *s);
 
+//! Use these functions to get the commonly used constants as basic.
+//! Assigns to s a SymEngine constant with name c
+void basic_const_set(basic s, char* c);
```

Comment:
Can you document here, that this function creates a new `SymEngine::constant` from a **copy** of the string in `c`, thus the caller is free to use `c` afterwards, and also the caller must free `c`.